static/app/views/issueDetails/allEventsTable.tsx

```diff
@@ -24,15 +25,35 @@ export interface Props {
   excludedTags?: string[];
 }
 
+const makeGroupPreviewRequestUrl = ({groupId}: {groupId: string}) => {
+  return `/issues/${groupId}/events/latest/`;
+};
+
 function AllEventsTable(props: Props) {
   const {location, organization, issueId, excludedTags, group} = props;
   const config = getConfigForIssueType(props.group);
   const [error, setError] = useState<string>('');
   const routes = useRoutes();
   const {fields, columnTitles} = getColumns(group, organization);
 
+  const endpointUrl = makeGroupPreviewRequestUrl({
+    groupId: group.id,
+  });
+
+  const {data} = useApiQuery<EventTransaction>([endpointUrl, {query: {}}], {
+    staleTime: 60000,
+  });
+
+  // TODO: this is a temporary way to check whether
+  // perf issue is backed by occurrences or transactions
+  // Once migration to the issue platform is complete a call to /latest should be removed
+  const groupIsOccurrenceBacked = !!data?.occurrence;
```

Comment:
This will always be `null` on first render and they may change to `true` after the request completes - is that a problem?

Do we want to return a loading spinner instead of `EventsTable` while the request above is completing?

Reply:
ah good point. I'm using both loading and error messages 🙏